src/app/pages/OpenWalletPage/webextension.tsx

```diff
@@ -1,16 +1,19 @@
 import React from 'react'
-import { useHref, useNavigate } from 'react-router-dom'
 import { openLedgerAccessPopup } from 'utils/webextension'
 import { FromLedger } from './Features/FromLedger'
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+import type { ExtensionRequestLedgerPermissionPopup } from '../../../../extension/src/ExtensionRequestLedgerPermissionPopup/ExtensionRequestLedgerPermissionPopup'
 
 export function FromLedgerWebExtension() {
-  const href = useHref('/open-wallet/connect-device')
-  const navigate = useNavigate()
+  /** See {@link ExtensionRequestLedgerPermissionPopup} */
+  const href = new URL(
+    '../../../../extension/src/ExtensionRequestLedgerPermissionPopup/index.html',
+    import.meta.url,
+  ).href
 
   return (
     <FromLedger
       webExtensionUSBLedgerAccess={() => {
-        navigate('/open-wallet/ledger/usb')
```

Comment:
uh I thought our mechanism to request ledger permissions in ext worked completely differently. It requires redux syncing from openLedgerAccessPopup? :O